src/Core/Framework/App/Event/PostAppDeletedEvent.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\App\Event;
+
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\Event\ShopwareEvent;
+use Shopware\Core\Framework\Log\Package;
+use Symfony\Contracts\EventDispatcher\Event;
+
+#[Package('framework')]
+class PostAppDeletedEvent extends Event implements ShopwareEvent
```

Comment:
the other events are just called `AppInstalledEvent` or `AppUpdatedEvent` why the post prefix? :grin:

Reply:
There is already a deleted event but it is fired before the app is actually deleted, so we can't really remove the files at that point (in rufus), as during the deleting the app files are read 😄